terraform/modules/lambda_edge/main.tf

```diff
@@ -0,0 +1,73 @@
+##################################################################################
```

Comment:
Can you rename the folder? lambda_edge is too generic - we'll have both origin request AND response Lambda@Edge